src/components/layout/SendTip.vue

```diff
@@ -41,22 +44,29 @@
   import util from "../../utils/util";
   import aeternity from "../../utils/aeternity";
   import { EventBus } from "../../utils/eventBus";
+  import avatar from '../../assets/userAvatar.svg';
 
   export default {
     name: 'SendTip',
     components: {
       FiatValue
     },
     computed: {
-      ...mapGetters(['balance', 'loading'])
+      ...mapGetters(['balance', 'loading']),
+      isDisabled(){
+        return this.sendTipForm.amount <= 0
+                || this.sendTipForm.url.length === 0 
+                  || this.sendTipForm.url.indexOf('http') !== 0
```

Comment:
I see it like (and reverse check for the button)
```suggestion
      isValid() {
        // TODO: better validation
        return this.sendTipForm.amount >= 0
            || this.sendTipForm.url.indexOf('http') === 0;
```

Reply:
Please add better/more descriptive naming to the functions.

Reply:
It might further migrate to `return this.getErrors() !== {};`
and getErrors() be like
```
let errors={};
if (this.sendTipForm.amount <=0) { this.errors['amount']['min'] = 'Positive number required' };
if (this.sendTipForm.url.regex(...) <=0) { this.errors['url']['regex'] = 'URL/AE address required' };
return errors;
```

Reply:
> Please add better/more descriptive naming to the functions.
IMHO ideally if we keep components to work with a single model, then methods become contextualized and self-explanatory (and shorter)

Reply:
@venimus when we start to engineer error handling we should think about a global solution for the product.

Reply:
For now it will be easier to just name the methods more understandable. Few more symbols wont hurt us.